api/Gemfile

```diff
@@ -61,6 +61,7 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rails-erd'
```

Comment:
Comment what this is for please.

Reply:
Do you mean comment on the file? Or do you mean comment on what it is used for? If it's the latter, I've already described the use in issue #123.

Reply:
Sorry, comment in the file similar to the annotations.

Reply:
I've added the annotations.